elements/pl-number-input/pl-number-input.py

```diff
@@ -183,21 +183,51 @@ def parse(element_html, data):
         data['format_errors'][name] = 'No submitted answer.'
         data['submitted_answers'][name] = None
         return
-
-    # Convert to float or complex
-    try:
-        a_sub_parsed = pl.string_to_number(a_sub, allow_complex=allow_complex)
-        if a_sub_parsed is None:
-            raise ValueError('invalid submitted answer (wrong type)')
-        if not np.isfinite(a_sub_parsed):
-            raise ValueError('invalid submitted answer (not finite)')
+        # Accept fractional input, then convert it to a float or decimal
+        # (only works for certain integer addition/subtraction problems)
+    elif '/' in a_sub:
+        a_sub = a_sub.strip()
+        a_sub_list = a_sub.split('/')
+        try:
+            a_sub_parsed_numerator = pl.string_to_number(a_sub_list[0])
+            if a_sub_parsed_numerator is None:
+                raise ValueError('invalid submitted answer (wrong type)')
+            if not np.isfinite(a_sub_parsed_numerator):
+                raise ValueError('invalid submitted answer (not finite)')
+                return
+        except Exception:
+            data['format_errors'][name] = 'Invalid format. The submitted answer could not be interpreted as a double-precision floating-point number or fraction.'
+            data['submitted_answers'][name] = None
+            return
+        try:
+            a_sub_parsed_denominator = pl.string_to_number(a_sub_list[1])
+            if a_sub_parsed_denominator is None:
+                raise ValueError('invalid submitted answer (wrong type)')
+            if not np.isfinite(a_sub_parsed_denominator):
+                raise ValueError('invalid submitted answer (not finite)')
```

Comment:
I'm having trouble following the Exception logic here -- it looks like we raise a ValueError with an error message, but then we catch it below and discard this error message?